etcdserver/server.go

```diff
@@ -326,6 +332,13 @@ func (s *EtcdServer) ID() types.ID { return s.id }
 func (s *EtcdServer) RaftHandler() http.Handler { return s.r.transport.Handler() }
 
 func (s *EtcdServer) Process(ctx context.Context, m raftpb.Message) error {
+	s.mu.Lock()
+	started := s.started
+	s.mu.Unlock()
+	if !started {
+		log.Printf("etcdserver: drop message from member %s because etcdserver has not been started", types.ID(m.From))
```

Comment:
i do not think the error message is useful to the end users. this is more like a very uncommon internal race.